src/components/ApplySection.tsx

```diff
@@ -6,7 +6,7 @@ function ApplySection(props: any) {
     <article className="applyPage__applicationContent" data-aos="fade-down">
       <li>
         <h3>({props.term})</h3>
-        <h2>{props.role}</h2>
+        <h2>({props.role})</h2>
```

Comment:
Any reason the curly braces were removed here but not in the previous line?

Reply:
I think that was a mistake, do I need to push new changes to revert it?

Reply:
You can push a new change to remove the outer braces, thanks